.github/workflows/ion-java-performance-regression-detector.yml

```diff
@@ -0,0 +1,61 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Ion Jave performance regression detector
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build-ion-java:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+
+
+        with:
+          fetch-depth: 2
+          submodules: recursive
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - run: git checkout HEAD^
+      - run: mvn clean install
+
+
+
+      - uses: actions/checkout@v2
+        with:
+          repository: amzn/ion-java-benchmark-cli
+          ref: master
+
+      - name: Build with Maven
+        run: mvn clean install
+      - name: Verify the CLI executes
+        run: java -jar target/ion-java-benchmark-cli-0.0.1-SNAPSHOT-jar-with-dependencies.jar --version
+      - name: clean maven repository
+        run : rm -r /home/runner/.m2
```

Comment:
> Is it possible to avoid deleting the whole directory?
> 
> E.g., The only different thing we have is different commits of ion-java, Is there anyway to replace ion-java only? This would make the workflow more efficient.

Is that mean replace the /.m2/repository/ion/ion-java?  When the first version of ion-java has been built, all dependencies relevant to this version of ion-java will be downloaded to /.m2/repository/ion/ion-java, the reason I deleted the whole directory is to make sure when the new commit come in, and build ion-java-benchmark-cli will not reuse the dependency from the last step. But yes, if only delete the .m2/repository/ion/ion-jave would be more efficient, that would be better.

Reply:
Discussed offline.